src/dashboard/components/navbar_component.py

```diff
@@ -65,7 +69,19 @@ def generate_navbar_items(
             class_name = NON_HIGHLIGHT_STYLE
 
         navbar_list.append(
-            dcc.Link(href=item["path"], children=item["name"], className=class_name)
+            dcc.Link(
+                href=item["path"],
+                className=f"{class_name}",
+                children=[
+                    html.Div(
+                        className="flex items-center space-x-4",
+                        children=[
+                            html.Img(src=get_asset_url(f'{item["name"].lower()}.svg')),
+                            html.P(item["name"]),
+                        ],
+                    )
+                ],
+            )
```

Comment:
Maybe extract generation of a single nav item to its own function?

Reply:
Its going to be a separate function later on when we add the functionality to add buttons